### PR TITLE
updated dropdown select all function for non-query options

### PR DESCRIPTION
**Pull request by kwongz**
### Checklist

- [ ] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x ] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [ ] I have added to the docs where applicable
- [ ] I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable

### CODE REVIEW

**Comment by kwongz** [comment]
Video Demo of select all with non-query options and select all with query + non-query options

https://www.loom.com/share/0140241c0c3848d08e1899abecac2c8a?sid=bf1dff11-c682-4cbb-81d1-c632da83992d

File: packages/ui/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte [hunk 1]
```diff
@@ -288,9 +286,17 @@
...
+			$queryOptions.forEach((opt) => {
+				flagOption([opt, DropdownValueFlag.FORCE_SELECT]);
+			});
+		}
+
+		if ($options) {
+			$options.forEach((opt) => {
+				flagOption([opt, DropdownValueFlag.FORCE_SELECT]);
+			});
+		}
```
Comment (ItsMeBrianD) [inline packages/ui/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte:299]: Is `$options` a superset of `$queryOptions`?

**Comment by kwongz** [comment]
resolves #2066

File: .changeset/fresh-bugs-rescue.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@evidence-dev/core-components': minor
```
Comment (ItsMeBrianD) [inline .changeset/fresh-bugs-rescue.md:2]:
```suggestion
'@evidence-dev/core-components': patch
```

**Review by ItsMeBrianD — APPROVED** [review]

**Pull request merged**